test/unit/RefRed/configuration/test_loading_configuration.py

```diff
@@ -167,7 +169,9 @@ def test_getMetadataObject(self):
             'to_lambda_range': 12.012,
             'data_sets': 'dataset1, dataset2',
             'tof_range_flag': 'tof_range_flag',
-            'norm_flag': 'norm_flag',
+            # Applying normalization is a global setting in the application,
+            # it should be treated that way when loading a template file.
+            # 'norm_flag': 'norm_flag',
```

Comment:
is this code won't every be used, I think it's better to remove it along with the comment

Reply:
Done